modules/view-images-in-your-environment.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images.adoc
```

Comment:
@gaurav-nelson thank you for your further updates. You could update the "Modules included in..." references to reflect the file name changes that you have made in this PR.  For example, this I think should now be:

`  // *  operating/examine-images-for-vulnerabilities.adoc`

You can automatically generate this header list for a module by running the `./scripts/list_assemblies_used_by_modules.sh` script in your local clone of the openshift-docs repository.

The same applies to the headers of the other modules in this PR, but I won't mention them all in this review.